RangeBarSample/src/main/res/layout/activity_recyclerview_adapter.xml

```diff
@@ -0,0 +1,36 @@
+<FrameLayout android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
```

Comment:
This file should be renamed to `recyclerview_item`

Reply:
Done.